app/models/loan.js

```diff
@@ -0,0 +1,20 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+var Types = Schema.Types;
+var ObjectId = Types.ObjectId;
+
+var QueryPlugin = require('mongoose-query');
+
+var LoanSchema = new Schema({
+  loan_date : { type:Date, required: true, default: Date.now},
+  loaner : { type:ObjectId, ref:'User', required: true },                  
+  items : [{item : { type:ObjectId, ref:'Item', required:true }, 
```

Comment:
please put `item..` to own row - more clear to read
there could be also optional reference to actual unique resource (`Resource` -model)